sdk/batch/azure-compute-batch/src/main/java/com/azure/compute/batch/implementation/BatchClientImpl.java

```diff
@@ -1169,7 +1169,7 @@ Response<BinaryData> listSubTasksInternalSync(@HostParam("endpoint") String endp
             Context context);
 
         @Post("/jobs/{jobId}/tasks/{taskId}/terminate")
-        @ExpectedResponses({ 200 })
+        @ExpectedResponses({ 204 })
```

Comment:
Swagger is 204, so I assume it is good.

https://github.com/Azure/azure-rest-api-specs/blob/main/specification/batch/data-plane/Azure.Batch/preview/2024-02-01.19.0/BatchService.json#L3128